docs/fr/documentation/schedule/reference.md

```diff
@@ -582,7 +582,7 @@
  
  Clé primaire (`shape_id`, `shape_pt_sequence`) 
  
- Les Tracé des lignes décrivent le chemin qu’un véhicule parcourt le long d’un l’alignement de l’itinéraire, et sont définis dans le fichier shapes.txt. Les Tracé des lignes sont associées aux Trajets et consistent en une séquence de points par lesquels le véhicule passe dans l’ordre. Les Tracé des lignes n’ont pas besoin d’intercepter exactement l’emplacement des Arrêts, mais tous les Arrêts d’un trajet doivent se trouver à une petite distance de la forme pour ce trajet, c’est-à-dire à proximité des segments de ligne droite reliant les points de la forme. 
+ Les Tracé des lignes décrivent le chemin qu’un véhicule parcourt le long d’un l’alignement de l’itinéraire, et sont définis dans le fichier shapes.txt. Les Tracé des lignes sont associées aux Trajets et consistent en une séquence de points par lesquels le véhicule passe dans l’ordre. Les Tracé des lignes n’ont pas besoin d’intercepter exactement l’emplacement des Arrêts, mais tous les Arrêts d’un trajet doivent se trouver à une petite distance de la forme pour ce trajet, c’est-à-dire à proximité des segments de ligne droite reliant les points de la forme. Le fichier shapes.txt doit être inclus pour tous les services basés sur l'itinéraire (pas pour les services à la demande basés sur les zones).
```

Comment:
Here the English is:

> The shapes.txt file should be included for all route-based services (not for zone-based demand-responsive services).

I'm surprised the _should_ got translated to _dois_ (which is must in _English_). 
In the glossary, _should_ should be translated into french to _devrait_, which would give this translation:

> Le fichier shapes.txt **devrait** être inclus [...]